platform/include/retry_utils.h

```diff
@@ -31,6 +31,141 @@
 /* Standard include. */
 #include <stdint.h>
 
+/**
+ * @page retryutils_page Retry Utilities
+ * @brief An abstraction of utilities for retrying with exponential back off and
```

Comment:
Are we wanting to call this as an _abstraction_? This is used only by the demos...

Reply:
And is provided as a guide for customers.